### ISSUE TITLE
Accessibility of multiple fields that are named the same

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#1096]
SODA flagged contact emails, contributors, keywords, and related works as having multiple form fields that are named the same. Alan suggested: 

- the modal subform itself might have repeated fields... Similar to GoogleScholar, the "Contributor name" field doesn't have a label 
See this ORCID example: 

<img width="674" alt="Image" src="https://github.com/user-attachments/assets/501b6499-2db0-4f69-8cb3-fc4fb962ae31" />

- Not sure exactly how it's working, but it appears to update the label with the text entered into the field ("Test1") so that it's not flagged as missing a label
- So to summarize what I think is going on here: any empty fields are unlabeled (axe Dev tools raises a missing label error). Once the user enters text into the field, that text becomes the field's label (no label error)

**Comment by justinlittman** [comment]
Other than it being reported by axe, is there any evidence that this is actually an accessibility problem? I'm very reluctant to start adding more JS just to check off an accessibility checkbox.

**Comment by thatbudakguy** [comment]
This is the issue identified by SODA as #2254, "Label does not convey purpose of control". It's come up a few times already; Astrid ticketed this so I could pursue a solution we discussed together with Alan.

**Comment by justinlittman** [comment]
I think we should have evidence that this is actually an accessibility problem before putting time into it / adding additional code.

**Comment by thatbudakguy** [comment]
Is SODA identifying it as "serious" not sufficient to make it an actual accessibility problem? Who else would we ask?

**Comment by justinlittman** [comment]
Yes, that is not sufficient. SODA has a track record of one person deciding something is "serious" and then another person deciding that no change is necessary. Therefore, I think it is important that we actually understand and be able to reproduce the accessibility problem. Is this a screen reader problem? If so, how does it sound in an actual screen reader?
